wp-content/themes/core/templates/archive.html

```diff
@@ -1,21 +1,31 @@
 <!-- wp:template-part {"slug":"header","tagName":"header","className":"site-header"} /-->
-<!-- wp:group {"tagName":"main","layout":{"type":"constrained"}} -->
-<main class="wp-block-group">
-	<!-- wp:query-title {"type":"archive"} /-->
-	<!-- wp:term-description /-->
-	<!-- wp:query -->
-	<div class="wp-block-query">
-	<!-- wp:post-template -->
-		<!-- wp:post-featured-image /-->
-		<!-- wp:post-title {"isLink":true} /-->
-		<!-- wp:post-author {"showAvatar":false} /-->
-		<!-- wp:post-date /-->
-		<!-- wp:post-terms {"term":"category"} /-->
-		<!-- wp:post-excerpt /-->
-	<!-- /wp:post-template -->
-	<!-- wp:template-part {"slug":"pagination"} /-->
-	</div>
-	<!-- /wp:query -->
-</main>
+<!-- wp:group {"tagName":"main","style":{"spacing":{"margin":{"top":"0","bottom":"0"},"padding":{"top":"var:preset|spacing|60","bottom":"var:preset|spacing|50"}}},"layout":{"type":"constrained"}} -->
+<main class="wp-block-group" style="margin-top:0;margin-bottom:0;padding-top:var(--wp--preset--spacing--60);padding-bottom:var(--wp--preset--spacing--50)"><!-- wp:query-title {"type":"archive","textAlign":"center","showPrefix":false,"align":"wide","style":{"spacing":{"margin":{"top":"0"}}}} /-->
```

Comment:
Throughout: I'd like to see us start using margin/padding block/inline in place of top/bottom & left/right.

Reply:
I'm not sure I follow. These are the parameters that get output from WordPress. I don't think we have control over the CSS properties used, right?

Reply:
Ah, duh. Ignore me.